rook/README.md

```diff
@@ -1,162 +1,160 @@
-# Rook installation
+# rook-ceph
 
-Also see the official documentation at <https://rook.io/docs/rook/v1.5/> and the repository at <https://github.com/rook/rook>.
+> **Note**: These instructions assumes that you are standing in this directory and have set `$CK8S_CONFIG_PATH` pointing to your environment.
 
-## Quickstart
+## Install
 
-```bash
-# Make sure your KUBECONFIG and Kubernetes context are set correctly
-./deploy-rook.sh
-```
+### Configure
 
-## Requirements
+The configuration is divided into an all cluster `commons` section and per cluster `clusters.<cluster>` section.
+Any configuration defined within the per cluster section will override anything set in the all cluster section.
+The name of the cluster will become the helmfile environment name.
 
-### Ceph partition or disk
+Many options use default within the helmfile state values.
 
-An empty disk or partition needs to be available.
-Ceph will create an [OSD](https://docs.ceph.com/en/latest/man/8/ceph-osd/) for each partition or disk to provide access to it.
+- Copy template
 
-To create an empty partition `/dev/vda2` on the `/dev/vda` disk, leaving 50 GB to other partitions, use the following cloud-init config:
+  ```bash
+  mkdir -p "${CK8S_CONFIG_PATH}/rook"
+  cp template/values.yaml "${CK8S_CONFIG_PATH}/rook"
+  ```
 
-```yaml
-#cloud-config
-bootcmd:
-- [ cloud-init-per, once, move-second-header, sgdisk, --move-second-header, /dev/vda ]
-- [ cloud-init-per, once, create-ceph-part, parted, --script, /dev/vda, 'mkpart 2 50GB -1' ]
-```
+- Set [preset](values/cluster-presets)
 
-### Nodes for Ceph mons
+  ```diff
+    cluster:
+      # see rook/helmfile.d/values/cluster-presets
+  -   preset: ""
+  +   preset: ceph-750
+  ```
 
-The Ceph [`mon`s](https://docs.ceph.com/en/latest/man/8/ceph-mon/) should not run on the same node.
-Make sure that there are at least as many worker nodes as there are replicas of `mon`s.
-See `spec.mon.count` in [cluster.yaml](./cluster.yaml).
+- Customising
 
-### Monitoring with Prometheus
+  Placement rules can be defined for all or per component, and resources can be overridden per component.
 
-Prometheus Operator CRDs should be installed before applying the Ceph cluster manifest if `spec.monitoring.enabled=true`.
+  Storage rules can be defined, the default is to use all devices on all nodes, and optionally it is possible to create a Block PVC based cluster for development.
+  (This requires special config for storage and mon.)
 
-## Installation
+### Deploy
 
-See [deploy-rook.sh](./deploy-rook.sh) for an example.
+The deployment is divided into two steps the bootstrap and the finalising.
+Since parts of the deployment relies on services in apps, such as Gatekeeper constraints and Prometheus operator, and apps relies on rook-ceph for block storage.
```

Comment:
```suggestion
Since parts of the deployment relies on services in [compliantkubernetes-apps](https://github.com/elastisys/compliantkubernetes-apps/), such as Gatekeeper constraints and Prometheus operator, and apps relies on rook-ceph for block storage.
```